### PR TITLE
Update to v47.0

### PR DESCRIPTION
**Pull request by bragefuglseth**

### CODE REVIEW

**Comment by bragefuglseth** [comment]
bot, do your thing

**Comment by bragefuglseth** [comment]
bot, build

**Comment by bragefuglseth** [comment]
bot, build

**Comment by bragefuglseth** [comment]
Now it should be ready to go once the GNOME 47 runtime is released.

File: org.gnome.Papers.json [hunk 1]
```diff
@@ -71,8 +71,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://poppler.freedesktop.org/poppler-24.06.1.tar.xz",
-                    "sha256": "1e629e8732286c745fbc0b15a3ee591443fb37a2210856e7f3ec38a0fb93ab13",
+                    "url": "https://poppler.freedesktop.org/poppler-24.07.0.tar.xz",
```
Comment (camelCaseNick) [inline org.gnome.Papers.json:74]: Is there a reason you chose 24.07 here? The automatically created #5 shows the latest version: 24.09.

**Comment by pabloyoyoista** [comment]
Thanks a lot for doing this! I'm quite surprised the 47 runtime is not yet ready, I guess release Team is busy :sweat_smile:

**Comment by camelCaseNick** [comment]
Why are you surprised, it always takes at least 3 days. It's usually available around Wednesday/Thursday.

**Comment by camelCaseNick** [comment]
The deadline for apps and for libraries is the same, because libraries can also contain translatable strings. So, not before three days before the general deadline. (This time between 12th and 14th.) Then, one SDK component misses the deadline, and with bad luck another doesn't play nicely with changes between freedesktop-sdk beta to release version, and then those three days are already over.

**Comment by pabloyoyoista** [comment]
bot, build

**Comment by bragefuglseth** [comment]
bot, build

**Comment by pabloyoyoista** [comment]
Thanks a lot @bragefuglseth, we can take updating poppler in a follow-up MR

**Pull request merged**